app/src/main/java/ly/loud/loudly/application/UpdateInfoService.java

```diff
@@ -220,7 +217,9 @@ private void restartUpdates() {
         if (loudlyPost == null) {
             return Observable.empty();
         }
-        return postsDatabaseModel.updateStoredInfo(loudlyPost, post.getNetwork(), diff);
+        return postsDatabaseModel
+                .updateStoredInfo(loudlyPost, post.getNetwork(), diff)
+                .onErrorReturn(error -> Info.emptyInfo());
```

Comment:
Here Info is difference between old info and new info. If we couldn't saved that, so let it be empty